api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -402,7 +419,9 @@ public ParticipantCohortAnnotation apply(org.pmiops.workbench.db.model.Participa
     org.pmiops.workbench.model.CohortReview responseReview = TO_CLIENT_COHORTREVIEW.apply(cohortReview, pageRequest);
     responseReview.setParticipantCohortStatuses(
       participantCohortStatuses.stream().map(TO_CLIENT_PARTICIPANT).collect(Collectors.toList()));
+    Timestamp now = new Timestamp(clock.instant().toEpochMilli());
 
+    userRecentResourceService.updateCohortEntry(cohort.getWorkspaceId(), userProvider.get().getUserId(), cohortId, now );
```

Comment:
I agree that this is the most logical place to update the cohort entry for cohort review.